src/did.ts

```diff
@@ -216,11 +216,19 @@ export class DID {
       // This version of the DID document has been revoked. Check if the JWS
       // was signed before it the revocation happened.
       const isEarlier = options.atTime && options.atTime < new Date(nextUpdate).valueOf()
-      if (!isEarlier) {
+      const isLater = !isEarlier
+      if (isLater) {
         // Do not allow using a key _after_ it is being revoked
         throw new Error(`JWS was signed with a revoked DID version: ${kid}`)
       }
     }
+    // Key used before `updated` date
+    const updated = didResolutionResult.didDocumentMetadata?.updated
+    const versionId = didResolutionResult.didDocumentMetadata?.versionId
+    const notV0 = versionId !== '0'
+    if (notV0 && updated && options.atTime && options.atTime < new Date(updated).valueOf()) {
```

Comment:
Were we not going to remove the versionId check here, as it's 3ID specific?

Reply:
Also what happens if updated is not defined here. Can be the case for various DID methods, e.g. Key DID.

Reply:
Ooh. Stupid me. Removed that in https://github.com/ceramicnetwork/js-did/pull/40

Reply:
> Also what happens if updated is not defined here. Can be the case for various DID methods, e.g. Key DID.

No `updated` ⇒ no reference timestamp ⇒ do not throw.

Reply:
I see, silly me :P